### ISSUE TITLE
Allow all objects to have instance variables including Touch and Keyboard

### ISSUE THREAD

**Issue by whscienceteacher** [Scirra/Construct-feature-requests#374]
### Reviewed guidelines

- [X] I have read and understand the suggestion guidelines

### Checked for duplicate suggestions

- [X] I checked for existing similar suggestions

### Summary

A few objects don't have instance variables, probably because they are objects that don't allow multiple instances. 

For code organization purposes, it would still be helpful for objects like touch, keyboard, and PlatformInfo to have instance variables, (or if you want name, it something different because they don't have instances that would be fine too.)

### Possible workarounds or alternatives

You can use global variables now, but it gets messy. Large projects can end up having huge numbers of global variables and splitting off TouchLastX, TouchLastY, etc. into the relevant object would be very helpful. 

### Proposed solution

Give all objects instance variables, even if there can only ever by one instance of the object. 

### Why is this idea important?

Large projects would benefit. 

### Additional remarks

I recognize this isn't the most important feature request and I know that you caution against saying that any request is easy. If this is difficult, please close it, but if it really is easy, it would be nice to have.

**Comment by F3der1co** [comment]
I agree, it would allow for having `musicVolume` and `sfxVolume` as variables on the Audio object for example, which makes a lot of sense for organization imo.
